### PR TITLE
Create controllers for end points

### PR DESCRIPTION
**Pull request by Pradhyumna02**
Move end points code to separate controllers and also added npm ignore

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.idea
+ansible
+config
```
Comment (rotosonic) [inline .npmignore]: For now we might have to include config folder for dynamo or write that module so we can pass in a config settings in a constructor or something like that

**Comment by rotosonic** [comment]
might want to update the name in package.json from bandwidth to comcastSpeedTest

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+CONTRIBUTING.md
+CONTRIBUTORS.md
+database.js
+Gruntfile.js
+LICENSE
+NOTICE
+package.json
+README.md
+Speed-testJS.iml
+Vagrantfile
```
Comment (rotosonic) [inline .npmignore]: new line needed

Comment (Pradhyumna02) [inline .npmignore]: updated

File: index.js [hunk 3]
```diff
@@ -25,215 +25,97 @@ var WebSocketServer = require('ws').Server;
...
-      var uploadtime = {'data':Date.now().toString()};
-      ws.send(JSON.stringify(uploadtime.data));
-    } else {
-      console.log("error message");
-    }
+    console.log('client connected');
+
+    ws.on('message', function incoming(messageObj) {
+        var message = JSON.parse(messageObj);
+        /*
```
Comment (kemarc) [inline index.js]: not sure why this is still here? could we just get rid of it?

**Comment by rotosonic** [comment]
1+ looks good

**Pull request merged**